### ISSUE TITLE
Create / improve translations

### ISSUE THREAD

**Issue by lamblinc** [xwikisas/application-urlshortener#3]
The "Get shortened URL" in page menu isn't translated, at least in French.

![image](https://github.com/xwikisas/application-urlshortener/assets/144363243/acbb757c-9b70-47a3-a124-9c88b70f8608)

Ticket created to track needed translations.

**Comment by oanalavinia** [comment]
Hi, @lamblinc !
Could you please help me with the french translation? There are only 3 keys, but since I know very little french, I might be  far away from the right formulation:) (I just used translate)

```
urlshortener.menu.button.title=Obtenir une URL raccourcie
urlshortener.menu.action.done=URL copiée dans le presse-papiers
urlshortener.menu.action.fail=Une erreur s'est produite lors de la création de l'URL raccourcie.
```

**Comment by oanalavinia** [comment]
Indeed, this app has only english translations. I created for now https://github.com/xwikisas/application-urlshortener/issues/6 for the french translation
